apps/block_scout_web/assets/webpack.config.js

```diff
@@ -175,7 +174,9 @@ const appJs =
       ),
       new ContextReplacementPlugin(/moment[\/\\]locale$/, /en/),
       new webpack.DefinePlugin({
-        'process.env.SOCKET_ROOT': JSON.stringify(process.env.SOCKET_ROOT)
+        'process.env.SOCKET_ROOT': JSON.stringify(process.env.SOCKET_ROOT),
```

Comment:
I'm not sure where these are used but this looks like environment vars must be set during compile time

Reply:
https://github.com/celo-org/celo-monorepo/pull/9810/files